docs/Migrating.md

```diff
@@ -9,6 +9,12 @@
   packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used
   to interact directly with a Unity environment, while `mlagents.trainers`
   contains the classes for training agents.
+* The supported Unity version has changed from `2017.1 or later` to `2017.4.10f1 or later`. The lower versions might still work, but you may encounter error that looks like this:
+```
+Instance of CoreBrainInternal couldn't be created. The the script class needs to derive from ScriptableObject.
+UnityEngine.ScriptableObject:CreateInstance(String)
+```
+You can fix the error by removing `CoreBrain` from the CoreBrainInternal.cs:16, click on your Brain Gameobject to let the scene recompile all the changed C# scripts, then add the `CoreBrain` back. (Make sure your brain is in Internal mode, and your TensorFlowSharp plugin is imported and ENABLE_TENSORFLOW flag is set), but this fix is only valid locally and unstable. 
```

Comment:
Cap line length to 80 characters (minus code samples and things that need to be verbatim).

Reply:
i would consider a few changes.  In migrating.md - indicate that we are now following Unity's LTS for support going forward in the future starting with 2017.4.. .  Indicate this helps us maintain good quality and support, etc..  and that future version of ML-agents using Unity's LTS versions will be supported but only in those future versions

 I would move the 2017.1 issue into either FAQ and provide a link to it (i.e. if you are still using older versions, see here for known issues in 0.5).  We should not necessarily call it out here

Lastly, I would add a note in the main read about our moving over to the LTS supported versions of Unity.

Reply:
@unityjeffrey 
1. What does "and that future version of ML-agents using Unity's LTS versions will be supported but only in those future versions" mean?
2. What is the "main read"？